### ISSUE TITLE
New file structure from demultiplex of NovaSeqX data

### ISSUE THREAD

**Issue by diitaz93** [Clinical-Genomics/cg#2208]
## Description

The V2 sample sheet used in the demultiplexing of NovaSeqX does not admit a sample project column where the ticket id was used previously. As a consequence, all the fastq files are saved in the flow cell directory without any grouping in sub-directories with the ticket id name.

## Suggested solution

SD: Change the logic in the demultiplex stats code to account for this change in directory structure

## This can be closed when

The change in file structure is no longer a problem.

## Blocked by

Finishing the implementation of manual demultiplex in cg.

**Comment by seallard** [comment]
@Vince-janv who is working on this?

**Comment by ChrOertlin** [comment]
This issue is already tracked in: #2168
